src/Adapters/Cache/RedisCacheAdapter.js

```diff
@@ -1,17 +1,18 @@
 import redis from 'redis';
 import logger from '../../logger';
 
-const DEFAULT_REDIS_TTL = 30 * 1000; // 30 seconds in milliseconds
+const DEFAULT_REDIS_TTL = 30; // 30 seconds in milliseconds
```

Comment:
@f0ster why the change here?  

https://github.com/f0ster/parse-server/blob/947471bc00c3b049278b61e0707b2f64cac13605/src/Adapters/Cache/RedisCacheAdapter.js#L50

we use psetex https://redis.io/commands/psetex

so if i've got it right, this change is not good?  cause we'll just cache for 30 millis by default.

PS sorry for the delay.